elements/pfe-tabs/demo/pfe-tabs.html

```diff
@@ -188,3 +161,36 @@ <h2 slot="header">Active Tab is Disabled</h2>
     <pfe-tab-panel>Aria Disabled</pfe-tab-panel>
   </pfe-tabs>
 </div>
+
+<div class="example">
+  <h2>Tabs first in markup</h2>
+  <pfe-tabs id="tabs-first">
+    <pfe-tab slot="tab">Users</pfe-tab>
+    <pfe-tab slot="tab">Containers</pfe-tab>
+    <pfe-tab slot="tab">Database</pfe-tab>
+    <pfe-tab slot="tab" disabled active>Disabled</pfe-tab>
+    <pfe-tab slot="tab" aria-disabled="true">Aria Disabled</pfe-tab>
+    <pfe-tab-panel>Users</pfe-tab-panel>
+    <pfe-tab-panel>Containers</pfe-tab-panel>
+    <pfe-tab-panel>Database</pfe-tab-panel>
+    <pfe-tab-panel>Disabled</pfe-tab-panel>
+    <pfe-tab-panel>Aria Disabled</pfe-tab-panel>
+  </pfe-tabs>
+</div>
+
+<div class="example">
+  <h2>Tabs with explicit relationships</h2>
```

Comment:
@hellogreg take a look at this particular case.

The internal plumbing is dependent on index, so the nth button always corresponds to the nth panel, regardless of pre-existing relationship in the markup.

So in your opinion should we
1. Be ok with this and merge - we don't need to support this case 
2. Merge now in order to unblock development but open a new issue to support this case
3. Block merging until this case is supported

Reply:
After talking with Steven about this, I say Let's Grab Two Martinis for now, but maybe revisit later.